question16/style.css

```diff
@@ -80,11 +81,43 @@ td {
 }
 .tabContents ul li a{
     text-decoration: none;
-    color: #3f51b5; 
+    color: #3f51b5;
+    position: relative;
+    line-height: 2;
 }
 .tabContents ul li a:hover{
     border-bottom: 1px solid #3f51b5;
 }
+.tabContents ul li a span.comment_icon_num {
```

Comment:
```suggestion
.comment_icon_num {
```

Reply:
I fixed it ! Thank you for poiting it out!

Latest class: https://github.com/csakyo/homework/commit/e507fac706dd73eee0409c7c71f3f465a3a636ce